akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKitScalaTestSpec.scala

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.testkit.typed.scaladsl
+
+import akka.actor.testkit.typed.TestKitSettings
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+import org.scalatest._
+import org.scalatest.concurrent.Eventually
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.Span
+
+/**
+ * A ScalaTest base class for the [[ActorTestKit]], making it possible to have ScalaTest manage the lifecycle of the testkit.
+ * The testkit will be automatically shut down when the test completes or fails using ScalaTest's BeforeAndAfterAll trait. If
+ * a spec overrides afterAll it must call super.afterAll.
+ *
+ * Note that ScalaTest is not provided as a transitive dependency of the testkit module but must be added explicitly
+ * to your project to use this.
+ */
+abstract class ActorTestKitScalaTestSpec(testKit: ActorTestKit) extends ActorTestKitBase(testKit)
```

Comment:
It is quite a long name, should we drop Spec at the end?

Reply:
yes

`ActorTestKitScalaTest` is also stacking a lot of words together and it might be difficult to see that it is about mixing ActorTestKit and ScalaTest. Perhaps `ActorTestKitWithScalaTest`, or `ScalaTestWithActorTestKit`.

The latter is perhaps more correct